tests/pallas/pallas_test.py

```diff
@@ -567,6 +567,56 @@ def test_pallas_call_block_shape_ndim_mismatch(self):
         "array shape"):
       f(a)
 
+  def test_pallas_call_input_output_aliases_errors(self):
+    x = np.arange(8 * 128, dtype=np.int32).reshape((8, 128))
+
+    # Invalid input index
```

Comment:
Up to you, but I'd drop the comments, because the regex matching the error message is self-explanatory.